plugins/GSdx/res/tfx.fx

```diff
@@ -918,6 +922,27 @@ PS_OUTPUT ps_main(PS_INPUT input)
 		if (c.a < 128.0f / 255.0f) c.a += 128.0f / 255.0f;
 	}
 
+	// Blending
+	// Accumulation blend + HDR algorithm. The only (partial) SW blending implemented on DX11 right now
+	if (PS_ACCU_BLEND == 1) // Cs*As + Cd or Cd - Cs*As
+	{
+		c.rgb = c.rgb * alpha_blend;
+	}
+	else if (PS_ACCU_BLEND == 2) // Cs*F + Cd or Cd - Cs*F
+	{
+		c.rgb = c.rgb * Af;
+	}
+
+	// Format to 16 bit color. This works for accu blend but may not in all cases.
+	// Needed for accurate shadows in Crash Tag Team Racing/Crash of the Titans
+	// FIXME: Maybe better to just enable for all 16 bit cases?
+	if (PS_ACCU_BLEND != 0 && PS_DFMT == FMT_16)
```

Comment:
I think you need to enable this when you have clamp 0 and accumulation blend only.

Reply:
Ok that works, it should still fix Tag Team Racing.

Reply:
Maybe it is not actually needed since we check 16bit format anyway.